### PR TITLE
[Unity] LookAt, take 3

### PR DESCRIPTION
**Pull request by eanders-MS**
Implements LookAtComponent.
Also cleaned up a few build warnings (non-XML).

### CODE REVIEW

File: MREUnityRuntime/MREUnityRuntimeLib/Util/Unity/AssetFetcher.cs [hunk 1]
```diff
@@ -8,9 +8,9 @@ namespace MixedRealityExtension.Util.Unity
 
     public static class AssetFetcher<T> where T : class
     {
-        public struct FetchResult<T>
+        public struct FetchResult<TT> where TT : T
```
Comment (stevenvergenz) [inline MREUnityRuntime/MREUnityRuntimeLib/Util/Unity/AssetFetcher.cs]: >TT [](start = 34, length = 2)

Did you mean to check this in? This seems unrelated to LookAt.

Comment (eanders-MS) [reply MREUnityRuntime/MREUnityRuntimeLib/Util/Unity/AssetFetcher.cs]: Fixing a build warning: T shadowed T from parent class.
